code-challenges.test.js

```diff
@@ -13,55 +18,161 @@
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("greaterThanTwo" , () => {
+  const fibLength1 = 6
+  const fibLength2 = 10
+  it ("returns numbers that are greater than 2", () => {
+    expect(greaterThanTwo(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
```

Comment:
I think this is missing an expect statement. You are provided 2 variables so we should expect 2 outputs